package/exe/exe.go

```diff
@@ -62,19 +76,25 @@ func isWaitError(err error) bool {
 }
 
 func (c *Cmd) Run() error {
-	return c.cmd().Run()
+	if err := (*exec.Cmd)(c).Run(); err != nil {
+		if canIgnore(err) {
```

Comment:
Tried to reduce the duplicate `signal: killed` logging. This will likely need to be revisited because I don't think the correct exit codes are being returned now under certain conditions.